src/Microsoft.Framework.DesignTimeHost.Abstractions/IPlugin.cs

```diff
@@ -9,5 +10,7 @@ namespace Microsoft.Framework.DesignTimeHost
     public interface IPlugin
     {
         void ProcessMessage(JObject data, IAssemblyLoadContext assemblyLoadContext);
+
+        Version Protocol { get; set; }
```

Comment:
Doesn't have to be. Thought I'd follow the same approach we used in SignalR since that proved to work well :smile: